jsonnet/build-jsonnet.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+set -e
+set -x
+# only exit with zero if all commands of the pipeline exit successfully
+set -o pipefail
+
+prefix="manifests"
```

Comment:
it seems like there are a few dependencies for this to run (jb, jq, jsonnet, mapfile, gojsontoyaml ) we should add them to the makefile.

Reply:
added dependencies like `jb`, `jsonnet` and `gojsontoyaml`. But, `mapfile` is an in-built bash command and `jq` [installation](https://stedolan.github.io/jq/download/) is a little bit tricky. How do you want to handle this?

Reply:
I think we can make a target for install-deps. We will mainly call it from our CI in which we control the platform

Reply:
So i'll go ahead and add target to install 64-bit jq binary?

Reply:
Yes, we will call it under travis-ci.